ui-tests/src/test/java/io/syndesis/qe/steps/other/DocsVerificationSteps.java

```diff
@@ -0,0 +1,46 @@
+package io.syndesis.qe.steps.other;
+
+import static org.junit.Assert.assertEquals;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$;
+
+import org.openqa.selenium.By;
+
+import com.codeborne.selenide.Selenide;
+
+import java.util.HashMap;
+
+import cucumber.api.java.en.Then;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class DocsVerificationSteps {
+    private HashMap<String, String> versions = new HashMap<>();
```

Comment:
this map can probably be a constant as it won't change so often. for initializing the map directly you can use `TestUtils.map()` method